python_modules/libraries/dagster-dg/dagster_dg/cli/utils.py

```diff
@@ -196,6 +196,28 @@ def _serialize_json_schema(schema: Mapping[str, Any]) -> str:
     return json.dumps(schema, indent=4)
 
 
+def validate_modules_accessible_or_err(project_context: DgContext) -> None:
+    executable_path = (
+        project_context.project_python_executable
+        if project_context.use_dg_managed_environment
+        else project_context.get_executable("python")
+    )
+    module_name = str(project_context.code_location_target_module_name)
+
+    python_stmt = f'import importlib; import sys; sys.exit(1 if importlib.util.find_spec("{module_name}") is None else 0)'
```

Comment:
This doesn't feel great, a more robust option might be to move this to a really simple `dagster-components` CLI command.